pom.xml

```diff
@@ -231,18 +231,6 @@
     </dependency>
 
     <!-- static analysis -->
-    <dependency>
-      <groupId>com.github.spotbugs</groupId>
-      <artifactId>spotbugs-annotations</artifactId>
-      <scope>provided</scope>
-      <optional>true</optional>
-      <exclusions>
-        <exclusion>
-          <groupId>com.google.code.findbugs</groupId>
-          <artifactId>jsr305</artifactId>
-        </exclusion>
-      </exclusions>
-    </dependency>
```

Comment:
As of 2.249, `jenkins-core` (which we depend on above) already depends on `spotbugs-annotations` transitively, so this is redundant. Furthermore, it causes Enforcer errors because of Maven's "nearest definition" algorithm. Better to remove it and have plugins get the version corresponding to their core baseline.